runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaCacheServerFactory.java

```diff
@@ -47,6 +49,8 @@ public final class KafkaCacheServerFactory implements KafkaStreamFactory
     private final Int2ObjectHashMap<BindingHandler> factories;
     private final Long2ObjectHashMap<KafkaBindingConfig> bindings;
     private final KafkaCacheServerAddressFactory cacheAddressFactory;
+    private final LongFunction<CatalogHandler> supplyCatalog;
+    private final ValidatorFactory validator;
```

Comment:
This is a validator factory, not an individual validator, so it might be better to name it `validators` or `validatorFactory` instead.

Later this will move to the `engine` and provided via a method on `EngineContext` instead of instantiating the factory directly.